native/src/components/TtsContainer.tsx

```diff
@@ -93,25 +89,20 @@ const TtsContainer = ({ children }: TtsContainerProps): ReactElement => {
     setIsPlaying(false)
   }
 
-  const playNext = useCallback(
-    (index = sentenceIndex) => {
-      const nextIndex = index + 1
-      if (nextIndex < sentences.length) {
-        setSentenceIndex(nextIndex)
-        play(nextIndex)
-      } else {
-        stop()
-      }
-    },
-    [play, sentenceIndex, sentences.length],
-  )
-
-  const playPrevious = (index = sentenceIndex) => {
-    const previousIndex = index - 1
-    if (previousIndex >= 0) {
-      setSentenceIndex(previousIndex)
-      play(previousIndex)
+  const playNext = useCallback(() => {
```

Comment:
@bahaaTuffaha for me it is definitely not necessary to pass the sentece index here.